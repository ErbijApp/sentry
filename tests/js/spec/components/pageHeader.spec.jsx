import React from 'react';
import {shallow} from 'enzyme';
import PageHeader from 'app/components/pageHeader';

describe('PageHeader', function() {
  it('renders', function() {
    let wrapper = shallow(<PageHeader>New Header</PageHeader>);
    expect(wrapper).toMatchSnapshot();
  });
});
