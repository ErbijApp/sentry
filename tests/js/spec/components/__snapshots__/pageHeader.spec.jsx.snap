// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader renders 1`] = `
<Wrapper>
  New Header
</Wrapper>
`;
